coq-stdlib.opam

```diff
@@ -29,11 +29,22 @@ depends: [
 build: [
   # Requires dune 2.8 due to https://github.com/ocaml/dune/issues/3219
   # ["dune" "subst"] {pinned}
+  #
+  # XXX need to run configure as in coq-core, or else dunestrap will
+  # use the default rule in config
+  [ "./configure"
```

Comment:
I'm confused. I thought configure was not needed by Makefile.dune?

Reply:
It is optional. If the developer doesn't run configure, we will use some defaults. That's convenient when developing, but doesn't produce correct packages as the prefix is not properly set.